lib/cart.rb

```diff
@@ -0,0 +1,24 @@
+require_relative "take_input"
+require_relative "item"
+require_relative "bill"
+
+class Cart
+  attr_reader :order
+
+  def initialize
+    @order = Hash.new(0)
+  end
+
+  def add_to_cart
+    puts "Order has been confirmed. Thank you.\n\n"
+    TakeInput.new.take_input(order)
+    cart_items
```

Comment:
Use proper names for functions. In this case, print_cart_items would have been better.